enterprise-suite/console_test/util/helm/helm.go

```diff
@@ -0,0 +1,65 @@
+package helm
+
+import (
+	"time"
+
+	apiv1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/lightbend/console_test/util"
+)
+
+const ServiceAccountName = "tiller"
+const TillerNamespace = "kube-system"
```

Comment:
I think this should be the same namespace as wherever we run the tests. E.g. we don't want to install tiller in kube-system on a shared cluster.

Again though - we could also just rip this out and let the upper layer scripts sort out helm, since it largely depends on environment.

Reply:
Reverted to using namespace provided by commandline flag with `lightbend-test` default.